README.md

```diff
@@ -1,76 +1,126 @@
 ---
 services: active-directory
-platforms: java
-author: navyasric
+platforms: Java
+author: sagonzal
+level: 300
+client: Java web application
+service: Microsoft Graph
+endpoint: AAD V1
 ---
+# Integrating Azure AD into a Java web application
 
-# Sign in using Azure AD account into a Java web application
+## About this sample
 
-This sample will give you a quick and easy way to set up authentication using Azure AD in a Java web application.
-The sample is designed to run on any platform and tested with Java 8.
+### Overview
 
-## Steps to run
+This sample demonstrates a Java web application calling a Microsoft Graph that is secured using Azure Active Directory.
 
-### Step 1: Register the sample with Azure AD
+1. The Java web application uses the Active Directory Authentication Library for Java(ADAL4J) to obtain a JWT access token from Azure Active Directory (Azure AD):
+2. The access token is used as a bearer token to authenticate the user when calling the Microsoft Graph.
 
-To use this sample you will need an Azure Active Directory tenant. If you're not sure what a tenant is or how you would get one, read [this document](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-howto-tenant).
+### Scenario
 
```

Comment:
What about having an image of the topology :-) ?

Reply:
Agreed, added one